book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] The same here. You should have created the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`. Kindly make sure A method for adding rentals is present.:


```ruby
 def add_rental(person, date)
    ...
  end
```


- Note:  the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.